networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -0,0 +1,36 @@
+// AWS Load Balancer Operator Release Notes
+:_content-type: ASSEMBLY
+[id="aws-load-balancer-operator-release-notes"]
+= AWS Load Balancer Operator release notes
+:context: aws-load-balancer-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The AWS Load Balancer (ALB) Operator deploys and manages an instance of the `aws-load-balancer-controller` resource.
+
+These release notes track the development of the AWS Load Balancer Operator in {product-title}.
+
+For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
+
+[id="aws-load-balancer-operator-release-notes-1.0.0"]
+== AWS Load Balancer Operator 1.0.0
+
+The following advisory is available for the AWS Load Balancer Operator version 1.0.0:
+
+// TODO: Replace "????" with the actual advisory once created.
+* link:https://access.redhat.com/errata/RHEA-2023:????[RHEA-2023:???? Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]
+
+[id="aws-load-balancer-operator-1.0.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the controller provisioned by the AWS Load Balancer Operator did not properly use the configuration for the cluster-wide proxy.
+These settings are now applied appropriately to the controller.
+
```

Comment:
Release 1.0.0 will have the new API version: `v1beta1`. Should we mention this too? I'm just not sure in which section.

Reply:
Yes, I believe we should mention this. I'd suggest a "Notable changes" section before the "Bug fixes" section for this information.